### DISCUSSION TITLE
Deploy to Computer Lab to all Users

### DISCUSSION THREAD

**Discussion by jim-davenport** [emoacht/Monitorian#552] [Q&A]
How would I go about deploying Monitorian to 25 PCs in a school computer lab, and have it load automatically for all users? Thanks!

**Comment by emoacht** [comment]
I am not familier with this kind of thing but this app's installer is a normal Windows Installer.

**Reply by jim-davenport** [reply]
I guess what I’m looking for is a switch that would enable it to start automatically on sign in. I have figured out how to deploy it to all users. Now I just need it to load on startup.

**Reply by emoacht** [reply]
As for the standard version, to enable to start on sign in, this app uses the registry of the system. See [Install/Uninstall](https://github.com/emoacht/Monitorian#installuninstall).
